Resources/Locale/en-US/_Impstation/spelfs/yes-and.ftl

```diff
@@ -37,8 +37,8 @@ spelf-mood-only-speak-to-command-desc = You are too important to speak to the ra
 spelf-mood-scheduler-name = Punctual
 spelf-mood-scheduler-desc = You believe that time must be strictly managed. Everything should be scheduled in advance, and tardiness is exceptionally rude.
 
-spelf-mood-radio-only-name = Impersonal
-spelf-mood-radio-only-desc = Speaking face-to-face is unacceptably personal. Any conversation must be had over the radio, or through an intermediary individual.
+spelf-mood-radio-only-name = Public Speaker
+spelf-mood-radio-only-desc = You firmly believe in the freedom of information. Speaking privately, face-to-face, is needlessly concealing information from the public. Your side of any conversation must be routed through a radio connection.
```

Comment:
little sad to see this one changed. I've had this mood a few times and I really enjoyed roping in unrelated people because I needed an intermediary and I couldn't use my radio